registry/src/config.rs

```diff
@@ -1 +1,14 @@
 // (c) Copyright 2023 Helsing GmbH. All rights reserved.
+use clap::Parser;
+use std::net::SocketAddr;
+
+#[derive(Parser, Clone, Debug)]
+pub struct Config {
```

Comment:
These should go in main imo, as they are binary related, the config.rs is usually for file based configs (at helsing)

Reply:
I'll put this into an `options.rs` instead then. Don't like having it in the `main.rs` for abstraction reasons and because I don't like importing a lot of types in `main.rs`.